services/wallet/service.go

```diff
@@ -175,19 +174,19 @@ func (s *Service) CheckConnected(ctx context.Context) *ConnectedResult {
 		}
 	}
 
-	opensea := make(map[uint64]Connection)
-	for chainID, client := range OpenseaClientInstances {
-		opensea[chainID] = Connection{
+	openseaClients := make(map[uint64]Connection)
+	for chainID, client := range opensea.OpenseaClientInstances {
+		openseaClients[chainID] = Connection{
 			Up:            client.IsConnected,
 			LastCheckedAt: client.LastCheckedAt,
 		}
 	}
 	return &ConnectedResult{
 		Infura:  infura,
-		Opensea: opensea,
+		Opensea: openseaClients,
 		CryptoCompare: Connection{
```

Comment:
Do we want to know the state per provider or per "category"? If it's the second, we might want to re-name the fields to something like "Blockchain", "Collectibles" and "Market"